dali/core/tensor_view_test.cc

```diff
@@ -345,5 +345,47 @@ TEST(TensorListViewTest, SampleRange) {
   }
 }
 
+TEST(TensorListViewTest, IsContiguous) {
+  TensorListView<StorageCPU, int, 3> tlv;
```

Comment:
How about testing `DynamicDimensions`? You you can have something like:
```
tlv.shape = {{
    { 1, 2, 3, 4 },
    { 10, 2, 1 },
    { 2, 3},
    { }
  }};
```

Reply:
That's not how DynamicDimensions works.